### DISCUSSION TITLE
next-i18next without internationalized routes

### DISCUSSION THREAD

**Discussion by rocramer** [i18next/next-i18next#1411] [General]
Is it possible to get this package working without using the internationalized routes feature in next.js? My users should change their language preference and this should not change the url.

Thanks!

**Comment by isaachinman** [comment]
The `next-i18next` package relies on NextJs for all routing. If you would like this feature, you should let the NextJs team know directly.

**Comment by dmi3y** [comment]
I use default and it works without internationalized routes.

UPD: clarification, you do not have to use explicit internationalized route

**Reply by jcjp** [reply]
> I use default and it works without internationalized routes.
> 
> UPD: clarification, you do not have to use explicit internationalized route

How did you do it without routing?

**Reply by carloschneider** [reply]
@jcjp any updates?

**Reply by jcjp** [reply]
> @jcjp any updates?

The library doesn't fit with our setup since we are using app routing vs pages routing, so I ended with a different solution that I found working for app routing.
